### PR TITLE
upgrade dependencies to sf5

### PR DESCRIPTION
**Pull request by GregoireHebert**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #61 
| License       | MIT

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -29,36 +29,36 @@
...
-        "symfony/dependency-injection": "^3.4 || ^4.2",
-        "symfony/event-dispatcher": "^3.4 || ^4.2",
-        "symfony/http-client": "^3.4 || ^4.2",
-        "symfony/http-foundation": "^3.4 || ^4.2",
-        "symfony/http-kernel": "^3.4 || ^4.2",
-        "symfony/options-resolver": "^3.4 || ^4.2",
-        "symfony/routing": "^3.4 || ^4.2",
-        "symfony/stopwatch": "^3.4 || ^4.2",
-        "symfony/translation": "^3.4 || ^4.3",
+        "symfony/config": "^4.2 || ^5.0",
```
Comment (vincentchalamon) [inline composer.json]: You should prefer `^4.4` as it's the LTS

File: composer.json [hunk 2]
```diff
@@ -29,36 +29,36 @@
...
+        "symfony/dependency-injection": "^4.2 || ^5.0",
+        "symfony/event-dispatcher": "^4.2 || ^5.0",
```
Comment (vincentchalamon) [inline composer.json]: Using the contract instead of the component?

File: features/Kernel.php [hunk 3]
```diff
@@ -76,8 +76,8 @@ protected function configureRoutes(RouteCollectionBuilder $routes): void
             $routes->import('@TwigBundle/Resources/config/routing/errors.xml', '/_error');
         }
 
-        $routes->import('.', null, 'docusign');
-        $routes->import('@TestBundle/Controller', null, 'annotation');
+        $routes->import('.', '/', 'docusign');
+        $routes->import('@TestBundle/Controller', '/', 'annotation');
```
Comment (GregoireHebert) [inline features/Kernel.php]: not in this PR, but I guess this might be an option.

Comment (vincentchalamon) [reply features/Kernel.php]: It's just related to e2e tests, it's not related to the final bundle source code

**Pull request merged**